packages/flutter/lib/src/material/text_field.dart

```diff
@@ -106,6 +106,7 @@ class TextField extends StatefulWidget {
     this.textCapitalization = TextCapitalization.none,
     this.style,
     this.textAlign = TextAlign.start,
+    this.textDirection = TextDirection.ltr,
```

Comment:
this should not have a default (or if it does, it should default to rtl)

Reply:
Wait, right to left? Why would that be?

Reply:
+1 for no default

Reply:
The time i did this PR i didn't know enumerations can have null value (I am a C# guy) and wished i could let the default value empty. This is the behavior i expected and i believe is correct.

@Hixie RTL default doesn't make any sense.

I am happy to see my PR is merged :)

Reply:
RTL makes as much sense as LTR, but it forces people to realize why it's a bad idea to have a default. See https://master-docs-flutter-io.firebaseapp.com/flutter/dart-ui/TextDirection-class.html for a more detailed discussion.